protos/flyteidl/admin/execution.proto

```diff
@@ -289,6 +290,13 @@ message ExecutionSpec {
 
     // Controls how to select an available cluster on which this execution should run.
     ClusterAssignment cluster_assignment = 20;
+
+    // Allows for the interruptible flag of a workflow to be overwritten for a single execution.
+    // Omitting this field uses the workflow's value as a default.
+    // As we need to distinguish between the field not being provided and its default value "false", we have to use a wrapper
+    // around the bool field. Since flyteadmin defines an interface requiring a unified interruptble override in several places, we cannot use
```

Comment:
can we omit the "Since flyteadmin defines an interface..." bit? I don't think we necessarily to dive into implementation details here. Also re: incompatible go types, couldn't we just have IsInterruptible return that generated oneof struct instead of a bool?

Reply:
Sure thing, will omit the extra comment 👍 

Re: incompatible go types - I'm not 100% sure we could wrap it again since the `GetInterruptible()` method used is auto-generated for the [`ExecutionSpec`](https://github.com/MorpheusXAUT/flyteidl/blob/execution-interruptible/gen/pb-go/flyteidl/admin/execution.pb.go#L1209) and [`LaunchPlanSpec`](https://github.com/flyteorg/flyteidl/blob/c412ba26cf64ed81d95b5aa786a2d477b83c0852/gen/pb-go/flyteidl/admin/launch_plan.pb.go#L467). As those are used directly, we'd have to wrap them again somehow to make them usable with `oneof`.
If you'd prefer that instead of using the `BoolValue` wrapper, which works for both cases, I can have another look tomorrow.